tensorflow_probability/python/bijectors/real_nvp_test.py

```diff
@@ -60,25 +64,39 @@ def testBijectorWithTrivialTransform(self):
       self.assertAllClose(x_, inverse_y_, rtol=1e-4, atol=0.)
       self.assertAllClose(ildj_, -fldj_, rtol=1e-6, atol=0.)
 
-  def testBijectorWithReverseMask(self):
-    flat_x_ = np.random.normal(0., 1., 8).astype(np.float32)
-    batched_x_ = np.random.normal(0., 1., (3, 8)).astype(np.float32)
-    num_masked = -5
+  @parameterized.parameters((-5, None), (None, -0.625))
+  def testBijectorWithReverseMask(self, num_masked, fraction_masked):
+    input_depth = 8
+    flat_x_ = np.random.normal(0., 1., input_depth).astype(np.float32)
+    batched_x_ = np.random.normal(0., 1., (3, input_depth)).astype(np.float32)
     for x_ in [flat_x_, batched_x_]:
       flip_nvp = tfb.RealNVP(
           num_masked=num_masked,
+          fraction_masked=fraction_masked,
           validate_args=True,
           shift_and_log_scale_fn=tfb.real_nvp_default_template(
               hidden_layers=[3], shift_only=False),
           is_constant_jacobian=False)
 
-      _, x2_ = np.split(x_, [8 - abs(num_masked)], axis=-1)
       x = tf.constant(x_)
 
-      # Check latter half is the same after passing thru reversed mask RealNVP.
       forward_x = flip_nvp.forward(x)
-      _, forward_x2 = tf.split(forward_x, [8 - abs(num_masked),
-                                           abs(num_masked)], axis=-1)
+
+      expected_num_masked = (
+          num_masked
+          if num_masked is not None
+          else np.floor(input_depth * fraction_masked))
+
+      self.assertEqual(flip_nvp._masked_size, expected_num_masked)
+
+      _, x2_ = np.split(x_, [input_depth - abs(flip_nvp._masked_size)], axis=-1)
+
+      # Check latter half is the same after passing thru reversed mask RealNVP.
+      _, forward_x2 = tf.split(
```

Comment:
Not sure about the linting here. Is there a command I can run to lint with the tfp styles?

Reply:
You can do `pylint --rcfile=testing/pylintrc <python files>`.

Reply:
It's still not happy about this line. I think you'll probably need to disable the lint here as it seems like a false positive. Just add a '  # pylint: disable=<lint name' here.